kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -202,18 +183,12 @@
 
   private static final String METRICS_JMX_PREFIX_DEFAULT_OVERRIDE = "kafka.rest";
 
-  /**
-   * <code>client.zk.session.timeout.ms</code>
-   */
-  public static final String KAFKACLIENT_ZK_SESSION_TIMEOUT_MS_CONFIG
-      = "client.zk.session.timeout.ms";
   public static final String KAFKACLIENT_TIMEOUT_CONFIG = "client.timeout.ms";
   /**
    * <code>client.init.timeout.ms</code>
    */
   public static final String KAFKACLIENT_INIT_TIMEOUT_CONFIG = "client.init.timeout.ms";
 
-  public static final String ZOOKEEPER_SET_ACL_CONFIG = "zookeeper.set.acl";
```

Comment:
This was unused, right?

Reply:
It was, no need to keep it around though right?

Reply:
Yes, we can remove.